client/src/modules/inventory/list/list.js

```diff
@@ -138,20 +140,30 @@ function InventoryListController($translate, Inventory, Notify, uiGridConstants,
   }
 
   function runResearch(params) {
+
     if (params) {
       search.assignFilters(params);
       vm.latestViewFilters = search.latestViewFilters();
       vm.hasCustomFilters = search.hasCustomFilters();
       vm.parameters = JSON.stringify(params);
     }
 
+    vm.loading = true;
+    vm.hasError = false;
+
     Inventory.read(null, params)
       .then(function (rows) {
         vm.gridOptions.data = rows;
       })
-      .catch(Notify.handleError);
+      .catch(Error);
+
+    vm.loading = false;
   }
 
+  function Error() {
```

Comment:
[Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) is a reserved word in JavaScript.  Could you name this function something else?